WBooks/Models/Book.swift

```diff
@@ -7,8 +7,21 @@
 
 import Foundation
 
-struct Book {
+public struct Book: Codable {
+    let id: Int
     let title: String
     let author: String
+    let genre: String
+    let year: String
+    let image: String
+    
 }
 
+enum BookKey: String, CodingKey {
+    case id = "id"
+    case title = "title"
+    case author = "author"
+    case genre = "genre"
+    case year = "year"
+    case image = "image"
+}
```

Comment:
I think this is not necessary since the coding keys are similar to the properties names by default. You can delete it if you want.